lib/ToggleButton/ToggleButtonAccessibilitySupport.js

```diff
@@ -0,0 +1,28 @@
+var
+	kind = require('enyo/kind');
+
+/**
+* @name ToggleButtonAccessibilityMixin
+* @mixin
+*/
+module.exports = {
+
+	/**
+	* @private
+	*/
+	observers: [
+		{method: 'updateAccessibilityAttributes', path: 'value'}
+	],
+
+	/**
+	* @private
+	*/
+	updateAccessibilityAttributes: kind.inherit(function (sup) {
+		return function (was, is, prop) {
+			var enabled = !this.accessibilityDisabled;
+			sup.apply(this, arguments);
+			this.setAttribute('aria-pressed', enabled ? String(this.value) : null);
+			this.setAttribute('aria-labelledby', enabled ? this.getId() : null);
```

Comment:
We will more consider whether or not we provide accessibilityLabeledBy mixin as public API. However, In ToggleButton case, we can set aria-labelledby to this.$.client.getId() because ToggleButton simply extends Button. We checked that screen reader(ChromeVox and TV) reads text within the inner MarqueeText well when we set aria-labelledby to this.$.client.getId() though this.$.client is aria-hidden:true